tests/test_filepath.py

```diff
@@ -18,34 +19,34 @@ def test_filepath(store="repo"):
 
     # create a mock file
     relpath = 'one/two/three'
-    os.makedirs(os.path.join(stage_path, relpath), exist_ok=True)
-    managed_file = os.path.join(stage_path, relpath, filename)
+    managed_file = Path(stage_path, relpath, filename)
```

Comment:
This test failed on WIN10. For example, the result of `managed_file` was `C:\\Users\\rguzman\\AppData\\Local\\Temp\\tmpb3gvp0ht\\one\two\threepicture.dat`.